website/docs/r/container_cluster.html.markdown

```diff
@@ -243,6 +260,10 @@ exported:
 * `instance_group_urls` - List of instance group URLs which have been assigned
     to the cluster.
 
+* `maintenance_policy.daily_maintenance_window.duration` - Duration of the time window, automatically chosen to be
```

Comment:
Nested fields are fun! This is actually `maintenance_policy.0.daily_maintenance_window.0.duration`

Reply:
Oh yes - I see this was recently updated for the master_auth stuff (this explains why it didn't work when I tried to use those attributes without the 0 index recently!)